lib/runtime/wazero/instance.go

```diff
@@ -36,13 +36,21 @@ const runtimeContextKey = contextKey("runtime.Context")
 
 var _ runtime.Instance = &Instance{}
 
+type cacheMetadata struct {
+	config      wazero.RuntimeConfig
+	cache       wazero.CompilationCache
+	hostModule  wazero.CompiledModule
```

Comment:
why do we need to store this?

Reply:
This is a nice find, i dont think we need to

Reply:
yeah, we should only keep the guest compiled module since this is the piece that are going to be instantiated on every exec.